tests/src/main/python/TestDriver_Json.py

```diff
@@ -63,9 +63,14 @@ def setup_class(self):
         global engineWrapper
         testWrapper = baseDir+os.environ['TEST_WRAPPER']
         cohortEngine = baseDir+os.environ['COHORT_ENGINE']
-        javabridge.start_vm(run_headless=True, class_path=javabridge.JARS + [cohortEngine, testWrapper]) # Start the JVM with modified classpath.
+        #  -Xms512M -Xmx1G -Djavax.net.ssl.trustStore=config/trustStore.pkcs12 -Djavax.net.ssl.trustStorePassword=change-password -Djavax.net.ssl.trustStoreType=pkcs12 
+        print("testwrapper: "+ testWrapper)
+        print("cohortEngine: "+ cohortEngine)
+        print("fhirServer: "+ os.environ['DATA_FHIR_SERVER_DETAILS'])
```

Comment:
The comment and these print statements should be removed.

Reply:
Removed Comment and Print statements.